build.gradle

```diff
@@ -29,18 +29,21 @@ repositories {
 }
 
 task('bootstrap') {
+  onlyIf {
+    project.hasProperty('g') && project.hasProperty('a') && project.hasProperty('d')
```

Comment:
Is property 'v' not needed?

Reply:
Not anymore, the version is pulled from the git tag.